report/4.Design/Performance.tex

```diff
@@ -0,0 +1,45 @@
+\section{Performance}
+In order to be able to track an object in real time, the object localization will have to happen in real time.
+Therefore, the algorithms should be tested in order to determine whether they will be able to execute the object localization fast enough for it to be feasible for the project.
+
+\subsection{Raspberry Pi}
+The Raspberry Pi 3B is the desired host of the MI processing, but it is very limited in the amount of resources available to it.
+It comes with a 1.2GHz Quadcore ARM Cortex-A53, 64Bit processor and 1 GB of RAM.
+In relation to computation power, the Raspberry Pi is a little underwhelming compared to an average consumer computer.
+Therefore, although an algorithm might execute fast enough on a normal computer, it is essential for it to be tested whether it will also be fast enough when executed on the Raspberry Pi.
```

Comment:
jo

```suggestion
Therefore, although an algorithm might execute quickly enough on a normal computer, it is essential for it to be tested whether it will also be quick enough when executed on the Raspberry Pi.
```